### DISCUSSION TITLE
Does the lemmatizer use info from NER?

### DISCUSSION THREAD

**Discussion by joakimwar** [explosion/spaCy#10296] [Help: Other Questions] [answered]
I know the lemmatizer uses information about the part of speech from the Tagger, but does it use information about the ent_type from the from NER at all? For example to not lemmatize words recognized as certain types of named entities? I can do this myself through 

`lemmas = [token.lemma_ if not token.ent_type else token.text for token in doc]`

but I was just wondering if any info about ent_type is already being utilized? I'm asking because I'm (pleasantly) surprised that certain named entities are not being lemmatized, even without me explicitly avoiding it.

**Comment by polm** [comment, marked as answer]
The lemmatizer does not use information from the NER component in any way. 

What is probably happening here is that your entities get the part of speech PROPN (proper noun), which means their lemmas don't change.

**Reply by joakimwar** [reply]
Thanks, that helps a lot!
